lldb/source/Core/Mangled.cpp

```diff
@@ -40,9 +43,10 @@ using namespace lldb_private;
 
 static inline bool cstring_is_mangled(llvm::StringRef s) {
   return Mangled::GetManglingScheme(s) != Mangled::eManglingSchemeNone
-         // BEGIN SWIFT
-         || SwiftLanguageRuntime::IsSwiftMangledName(s);
-         // END SWIFT
+#ifdef LLDB_ENABLE_SWIFT
+         || SwiftLanguageRuntime::IsSwiftMangledName(s)
+#endif
```

Comment:
I guess we'll eventually need some form of language runtime registry...

Reply:
`// LLDB_ENABLE_SWIFT`